moveit_planners/chomp/chomp_interface/test/rrbot_chomp_planning_pipeline.launch.xml

```diff
@@ -0,0 +1,14 @@
+<launch>
+	<!-- CHOMP Plugin for MoveIt! -->
+	<arg name="planning_plugin" value="chomp_interface/CHOMPPlanner" />
+
+	<arg name="start_state_max_bounds_error" value="0.1" />
+
+	<param name="planning_plugin" value="$(arg planning_plugin)" />
+	<param name="start_state_max_bounds_error" value="$(arg start_state_max_bounds_error)" />
+
+	<param name="collision_detector" value="Hybrid" />
```

Comment:
"Hybrid" is a bit non-descriptive but maybe we can leave that to another PR

Reply:
We should add this to a CHOMP issue list and work on it one issue at a time.

Reply:
+1 (as well to the point below) This is a huge improvement and should not be held back any longer!